man/useradd.8.xml

```diff
@@ -157,10 +157,13 @@
 	    user's login directory. The default is to append the
 	    <replaceable>LOGIN</replaceable> name to
 	    <replaceable>BASE_DIR</replaceable> and use that as the
-	    login directory name.  If the directory
-	    <replaceable>HOME_DIR</replaceable> does not exist, then
-	    it will be created unless the <option>-M</option> option
-	    is specified.
+	    login directory name.  
+	    The directory <replaceable>HOME_DIR</replaceable> is not created by
+	    default.  It will be created for non-system users if either the
+	    <option>-m</option> flag is specifed or
+	    <replaceable>CREATE_HOME</replaceable> in
+	    <filename>login.defs</filename> is set to true.  However will never
```

Comment:
I'm not completely sure, but shouldn't it be `However it will`?

Reply:
By the way, I am not a native English speaker so I may be wrong

Reply:
It would be clearer, yes, thank you.